resources/devices/mobile-device/unknown.json

```diff
@@ -11,5 +11,18 @@
             "isMobileDevice": "true",
             "isTablet": "false"
         }
+    },
+    "general Mobile Device (Touch)": {
```

Comment:
I wanted to add this device because I think it makes more sense to use this as the base device when the device isn't known (for Android mainly). `general Mobile Phone (Touch)` is the one used in this place currently, but I feel that it implies that the device is a phone (when it could be a tablet).  This is more ambiguous, so could mean both.